### PR TITLE
fix(multi): item rule has custom checker will throw error if validate…

### PR DESCRIPTION
**Pull request by 0x0a0d**
… failed

custom checker push errors to fnCustomErrorsX that will be assigned to var errors, not _errors as expected
To avoid a breaking changes, I just keep size of errors before validate and restore it if failed
But I think the logic should be changed
image

### CODE REVIEW

**Comment by 0x0a0d** [comment]
I add some test that will break with current master branch

File: test/rules/multi.spec.js [hunk 1]
```diff
@@ -88,4 +88,74 @@ describe("Test rule: multi", () => {
...
+		it("test num99", () => {
+			{
+				const o = { a: 99 };
+				expect(check(o)).toBe(true);
+				expect(o).toStrictEqual({ a: 99 });
+				expect(checkerFn).toBeCalledTimes(5);
+			}
+			{
+				const o = { a: 1199 };
+				expect(check(o)).not.toBe(true);
```
Comment (icebob) [inline test/rules/multi.spec.js:155]: Same

**Review by icebob — CHANGES_REQUESTED** [review]

**Comment by 0x0a0d** [comment]
@erfanium pls fix this too

**Review by icebob — APPROVED** [review]
Thanks!

**Pull request merged**